### ISSUE TITLE
unclear what list of people at the bottom is

### ISSUE THREAD

**Issue by michielbdejong** [solid/meeting-pane#15]
Is it a list of people who have opened the meeting with a data browser at least once? Should add some clarifying text there.

**Comment by timbl** [comment]
Yes it is.  Yes there should be.   In the code they are called Participants.  It only records their participation if it has the access to do so.
